.github/workflows/test-application.yaml

```diff
@@ -72,21 +66,15 @@ jobs:
                       ${{ runner.os }}-composer-
 
             - name: Install dependencies
-              run: |
-                  composer validate --strict
-                  composer update ${{ matrix.composer-flags }}
+              run: composer update ${{ matrix.composer-flags }}
 
-            - name: Create database
-              run: |
-                  Tests/app/console doctrine:database:create
-                  Tests/app/console doctrine:schema:update --force
-              env: ${{ matrix.env }}
+            - name: Bootstrap test environment
+              run: composer bootstrap-test-environment
 
             - name: Lint code
               if: ${{ matrix.lint }}
-              run: ./Tests/app/console lint:twig Resources/views --env test
-              env: ${{ matrix.env }}
+              run: composer lint
```

Comment:
Personally I would also add the `matrix.env` to `Lint code` and `Bootstrap test environment` for further cases, thus we just have to add the environment variable, WDYT ?